RealmSwift/RealmCollection.swift

```diff
@@ -530,6 +530,103 @@ public protocol RealmCollection: RealmCollectionBase {
     func thaw() -> Self?
 }
 
+// MARK: Aggregatable
+
+/**
+ Protocol for RealmCollections where the Value is of an Object type that
+ enables aggregatable operations.
+ */
+public protocol Aggregatable: RealmCollection where Element: ObjectBase {
```

Comment:
Why does this need to be a new protocol instead of just a conditional extension on RealmCollection?

Reply:
It made more sense to me when I was planning on deprecating the string based methods, but since we've reversed that, I agree that it no longer makes sense.